### ISSUE TITLE
Training to get paragraph vectors D

### ISSUE THREAD

**Issue by chenheh** [klb3713/sentence2vec#10]
Hi there,

I've read the paper by Quoc Le and Tomas Mikolov.
It seems that, at the first stage of the algorithm, what it does is training to get word vectors W, softmax weights U, b and paragraph vectors D on already seen paragraph.

But in your demo.py script, at stage one, it trains to get word vectors W, softmax weights U and b, using Word2Vec. It doesn't get the paragraph vectors D.
At the second stage of your demo.py script, it uses the word vectors obtained in stage one to get paragraph vectors D for new paragraphs.

It seems what your scripts do is not what the paper says.

Correct me if I am wrong.
Thanks.

He Chen

**Comment by mug2mag** [comment]
@chenheh  hi, have you know how to get paragraph vectors D now ? I really want to know how. But from the tutorial,  the Paragraph Vector frame just can get the word vectors like word2vec.  
Very appreciate it.
